src/core/application/tag/index.ts

```diff
@@ -0,0 +1,2 @@
+export { Tag } from './tag';
+export type { TCreateTagInput, TDeleteTagInput } from './tag.types';
```

Comment:
```suggestion
export * from './tag';
export * from './tag.types';
```
Can we export like that?